ChangeLog

```diff
@@ -36,7 +36,11 @@ Release date: TBA
 
 * bad-continuation and bad-whitespace have been removed, black or another formatter can help you with this better than Pylint
 
-Close #246, #289, #638, #747, #1148, #1179, #1943, #2041, #2301, #2304, #2944, #3565
```

Comment:
By the way these new Changelog entries should be moved under the 2.6 release, since they are now under 2.5.1.